### PR TITLE
Bot configuration - updating the code with latest SDK

### PR DESCRIPTION
**Pull request by Jegadeesh-MSFT**

### CODE REVIEW

File: samples/bot-configuration-app/nodejs/teamsBot.js [hunk 1]
```diff
@@ -1,281 +1,194 @@
...
+        },
+      },
+  };
+ 
+    /*
+      Option 2: You can add a "config/continue" response as below code
+      */
+    //   const adaptiveCard = CardFactory.adaptiveCard(this.adaptiveCardForContinue());
+    //   response = {
+    //       config: {
```
Comment (Pawank-MSFT) [inline samples/bot-configuration-app/nodejs/teamsBot.js:65]: use Multi-line comments.

**Review by Pawank-MSFT — COMMENTED** [review]
One small comment

File: samples/bot-configuration-app/nodejs/teamsBot.js (hunk 1 above)
Comment (Jegadeesh-MSFT) [reply samples/bot-configuration-app/nodejs/teamsBot.js]: Fixed

File: samples/bot-configuration-app/nodejs/teamsBot.js
Comment (deekshanya) [inline samples/bot-configuration-app/nodejs/teamsBot.js:1]: I would prefer that option 1 is commented out, while option 2 is not.
also some indentation fixes are needed

File: samples/bot-configuration-app/nodejs/README.md [hunk 2]
```diff
@@ -125,7 +125,16 @@ You can interact with this bot in Teams by sending it a message, or selecting a
...
   ![groupChat-AddedBot ](Images/groupChat-AddedBot.png)
 
-   **Show configurable card interaction:**
+   **Option 1 : Auth card Interactions:**
+   - Comment out option 2 code in [teamsBot.js](/samples/bot-configuration-app/nodejs/teamsBot.js) to experience the `config/Auth` feature.
+   - Note: The `Sign-in` page displayed is currently using a dummy URL. Please update it with your authentication URL.
+
+  ![Bot-description-card](Images/Bot-description-card.png)
+
+   **Option 2 : Bot Descrption Card:**
```
Comment (deekshanya) [inline samples/bot-configuration-app/nodejs/README.md:134]: nit: description

**Review by deekshanya — CHANGES_REQUESTED** [review]

File: samples/bot-configuration-app/nodejs/teamsBot.js
Comment (Jegadeesh-MSFT) [reply samples/bot-configuration-app/nodejs/teamsBot.js:1]: Fixed

File: samples/bot-configuration-app/nodejs/README.md (hunk 2 above)
Comment (Jegadeesh-MSFT) [reply samples/bot-configuration-app/nodejs/README.md]: Fixed

**Review by Prithvi-MSFT — APPROVED** [review]

**Review by deekshanya — APPROVED** [review]

**Comment by Jegadeesh-MSFT** [comment]
image
Build Pipeline executed

https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/8060361180

**Review by Pawank-MSFT — APPROVED** [review]
looks correct, Approving!

**Pull request merged**